dash-renderer/package.json

```diff
@@ -25,6 +25,7 @@
     "cookie": "^0.4.0",
     "dependency-graph": "^0.9.0",
     "prop-types": "15.7.2",
+    "fast-isnumeric": "^1.1.3",
```

Comment:
Given the numerous changes in `package-lock.json`, I assume that at some point `npm install` was run from a clean slate. To have a fully consistent dev/prod environment with minimal changes, would be preferable to revert the lock file, `npm ci` + `npm install --save-dev npm-run-all` + `npm install --save fast-isnumeric` and recommit the newly updated lock file with limited changes. Alternatively, if we are fully confident that these version changes do not impact Dash's behavior, they can be accepted as the new baseline.

Reply:
good call - not sure how that happened... -> https://github.com/plotly/dash/pull/1103/commits/3f46fe658dd48d17ece406d88015798291b5c6cb